apps/editing-toolkit/editing-toolkit-plugin/help-center/src/config.js

```diff
@@ -44,6 +44,8 @@ window.configData = {
 	},
 	signup_url: '/',
 	wapuu: false,
+	wpcom_signup_id: '39911',
```

Comment:
What is this for?

Reply:
@alshakero asked me to add the config as it might be that other flows in localhost might fail